openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -93,15 +93,15 @@ class MfgInspector(object):
     my_custom_converter)
   my_tester.add_output_callbacks(interface.save_to_disk(), interface.upload())
 
-  **Important** the conversion of the TestRecord to protofbuf as specified in
+  **Important** the conversion of the TestRecord to protobuf as specified in
   the _converter callable attribute only occurs once and the resulting protobuf
   is cached in memory on the instance.
 
   The upload callback will upload to mfg-inspector.com using the given
   username and authentication key (which should be the key data itself, not a
   filename or file).
 
-  In typical productin setups, we *first* save the protobuf to disk then attempt
+  In typical production setups, we *first* save the protobuf to disk then attempt
```

Comment:
Would you mind tweaking this to get every line to fix into 80 characters?

Reply:
Sure, can we write it as "prodn" or should I leave it as it was?

Reply:
Would you review the change I made addressing to the comment?

Reply:
Ah sorry I should have been more clear. Lets move 1 word down one line at a time here. Basically what I mean is change this paragraph to be:


```
  In typical production setups, we *first* save the protobuf to disk then
  attempt to upload the protobuf to mfg-inspector. In the event of a network,
  outage the result of the test run is available on disk and a separate process
  can retry the upload when the network is available.
```

Reply:
Thank You I changed it as desired